src/common/crypto_ope.h

```diff
@@ -14,8 +14,16 @@
 /** Length of OPE key, in bytes. */
 #define OPE_KEY_LEN 32
 
-/** Largest value that can be passed to crypto_ope_encrypt() */
-#define OPE_INPUT_MAX 131072
+/** Largest value that can be passed to crypto_ope_encrypt().
+
+ *  The current max value (220k seconds) stands for about 61 hours. The
+ *  rationale here is as follows: The rev counter is the time of seconds since
+ *  the start of an SRV period. SRVs are useful for about 48 hours (that's how
+ *  long they stick on the consensus). Let's also add 12 hours of drift for
+ *  clock skewed clients stuck with an old consensus, and we arrive to 60
+ *  hours.
```

Comment:
Hmm, that comment might have been a bit misleading. I meant to say "clock
skewed services" and not "clock skewed clients". I pumped it to 60 hours
because I'm concerned about clock skewed services using outdated consensus. In particular, an SRV stays for 48 hours in consensuses, and a service could potentially have an outdated consensus and keep on using it.

In general, 60 hours VS 48 hours doesn't make a huge difference in terms of
OPE, so perhaps we can leave it at 60 and be on the safe side.